### DISCUSSION TITLE
Self Hosting Renovate with Docker Issue

### DISCUSSION THREAD

**Discussion by tnbach3chan** [renovatebot/renovate#16531] [Request Help]
Hi everyone, I am trying to run self-hosted Renovate. When I tried to run 
`docker run --rm -v "/path/to/config.js:/usr/scr/app/config.js" renovate/renovate`
with this config.js file 
```
module.exports = {
    endpoint: 'https://api.github.com/',
    token: '<PAT>',
    platform: 'github',
    logLevel: 'debug',
    onboardingConfig: {
        extends: ['config:base'],
    },

    repositories: ['usr/repoName'],
    renovateFork: true,
    gitAuthor: "GIT_AUTHOR",
    username: "USR_NAME",
    onboarding: false,
    printConfig: true,
    requireConfig: false,
};
```
it showed:
```
FATAL: You must configure a GitHub personal access token
 INFO: Renovate is exiting with a non-zero code due to the following logged errors
       "loggerErrors": [
         {
           "name": "renovate",
           "level": 60,
           "logContext": "havr0-DwyLDLkaCThALqQ",
           "msg": "You must configure a GitHub personal access token"
         }
       ]
```
I also tried
`docker run --rm -e LOG_LEVEL="debug" -e GITHUB_COM_TOKEN="<PAT>" -v "/path/to/config.js:/usr/scr/app/config.js" renovate/renovate`
and looked other discussion but I still yield the same error.

How can I resolve this problem?

**Comment by rarkins** [comment]
You need to get debug logging working first. It must be defined in env. You seem to using some old config names, so is this not your first time using renovate?

**Reply by tnbach3chan** [reply]
I figured it out that I need to place the config.js file inside my repository. I have been able to start it up. Thank you for your support!
